easybuild/easyblocks/s/suitesparse.py

```diff
@@ -94,8 +94,12 @@ def configure_step(self):
         try:
             for line in fileinput.input(fp, inplace=1, backup='.orig'):
                 for (k, v) in cfgvars.items():
-                    line = re.sub(r"^(%s\s*=\s*).*$" % k, r"\1 %s # patched by EasyBuild" % v, line)
-                    if k in line:
+                    orig_line = line
+                    line = re.sub(r"^\s*(%s\s*=\s*).*$" % k,
```

Comment:
No, for the regex substitution, maybe just make the existing `patch file` comment a bit clearer, to explain what the regex in the for loop does?